Importer/Interpreter/Field/BooleanFieldInterpreter.php

```diff
@@ -43,7 +43,7 @@ public function interpret(FieldConfigurationInterface $configuration, $value)
             throw new \InvalidArgumentException();
         }
 
-        if (null === $value) {
+        if (null === $value || strlen(trim($value)) === 0) {
```

Comment:
Adding this condition changes the result for `false` value.
Previously the result was `false`, not it will be `null`.
Which one is correct?